### ISSUE TITLE
How do attributeSelectors work? (unprefixed class not being replaced)

### ISSUE THREAD

**Issue by j-greig** [JPeer264/node-rcs-core#119]
I'm trying to map the custom class substitutions below but cannot get `.center` to be replaced for some reason...

```
selectors: {
    '.pv5-m': 'py-4',
    '.w-100-m': 'w-full-m',
    '.center': 'mx-auto',
  },
  ```
  
  The original HTML markup is:
  ```<div class="w-80 w-90-xl w-100-m center pv6 pv5-m pv4-s ph4 ph3-s">```
  
  Why is the .center not mapped to .mx-auto?

**EDIT: realised that 'centre' is in the excludeList, is there a safe way to include/allow certain CSS class names on a case-by-case basis?

**Comment by JPeer264** [comment]
Hey @j-greig, thanks for the issue. Yes `.center` is in the excluded list, I didn't have time to update the README yet. There is a bigger list of caveats which should be read ([here](https://github.com/JPeer264/node-rename-css-selectors/blob/master/docs/caveats.md)).

There is no such way right now, but this might be of course a great addition. I think it would be also nice to give a warning in case a selector is ignored but found. 

I have time tomorrow and can implement these things

Checklist for myself:
- [ ] Add caveats into rcs-core's README
- [ ] Print out warnings in case an ignored selector has been found
- [ ] Allow items in case something has been put into the denylist

Will those changes work for your usecase?

**Comment by j-greig** [comment]
Sounds perfect @JPeer264 ! Thank you :)

(I somehow missed the caveats before)

**Comment by JPeer264** [comment]
Closed with #121

**Comment by JPeer264** [comment]
Released with v`3.6.0`

- To get a warning just add `rcs.warnings.warn()` after you are done with everything (as described in the README) 
- You can also include specifics with `rcs.selectorLibrary.setInclude('center')` **before** you set any selectors 

If this won't fix your issue feel free to reopen, or ask is something is unclear
